source/content/guides/decoupled/overview/03-site-options.md

```diff
@@ -64,6 +64,53 @@ The table below provides compatibility information for Front-End Sites.
 | WordPress  |   5.9 or higher |
 
 
+## Rendering Options
+
+This section provides a comparison of static site generation (SSG), server side rendering (SSR), and incremental static regeneration (ISR) modes, and some common use cases for each.
+
+Refer to [Decoupled Architectures: What Computer Assembles the Websites](https://pantheon.io/blog/decoupled-architectures-what-computer-assembles-websites) for more information. You can also check out this [Decoupled Architecture video](https://www.youtube.com/watch?v=dF39cXW3IqY) for a demonstration of the different approaches each mode takes.
+
+### What Is Static Site Generation (SSG)?
+
+SSG compiles and renders your website at build time. SSG output consists of:
+
+- **Static files:** HTML
+- **Assets:** JavaScript, CSS, etc.
+
+#### Common SSG Use Cases
+
+- Pages with content that is the same for all users
+- Pages that can be publicly cached
```

Comment:
```suggestion
- Websites with regular content changes/edits benefit from SSR since they do not need to compile the entire website at once like they would in SSG.
```